.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous Integration
+
+on:
+  push:
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write  # For removing tags from the PR
```

Comment:
These are not needed if you are only using github action machine and doesn't utilize aws credentials